src/CommonCartridge.js

```diff
@@ -618,8 +616,35 @@ export default class CommonCartridge extends Component {
 
                         <Route
                           exact
-                          path="/external/tool"
-                          render={({ match }) => <PreviewUnavailable />}
+                          path="/external/tool/:identifier"
+                          render={({ match, location }) => (
+                            <React.Fragment>
+                              {this.activeNavLink === undefined &&
```

Comment:
@RobertDHanna we can get rid off `this.activeNavLink` method, It was used to keep track from where user was coming from (from the navLink, since I wasn't able to get the location from the router when doing the back to all items button)

Reply:
@mauriciozaffari ok cool! I will do that. Question: do we want only remove `this.activeNavLink === undefined` but leave `!showcaseSingleResource && this.setActiveNavLink("/")` ?

Reply:
(the rest of the condition below `this.activeNavLink === undefined`

Reply:
we can remove the hole condition: 
`{this.activeNavLink === undefined &&
   !showcaseSingleResource &&
    this.setActiveNavLink("/")}`

or if you want I can remove that in other PR, then I'll remove others references to that method as well

Reply:
ah, ok - sounds good. I'm good to remove it since it is part of changing the nav buttons. thanks! I'll push an update with that removed (as well as in the other routes)